ci-operator/step-registry/cucushift/installer/check/cluster-health/cucushift-installer-check-cluster-health-commands.sh

```diff
@@ -163,6 +163,67 @@ function wait_clusteroperators_continous_success() {
     fi
 }
 
+function check_mcp() {
+    local updating_mcp unhealthy_mcp
+
+    updating_mcp=$(oc get mcp -o custom-columns=NAME:metadata.name,CONFIG:spec.configuration.name,UPDATING:status.conditions[?\(@.type==\"Updating\"\)].status --no-headers | grep -v "False")
+    if [[ -n "${updating_mcp}" ]]; then
+        echo "Some mcp is updating..."
+        echo "${updating_mcp}"
+        return 1
+    fi
+
+    # Do not check UPDATED on purpose, beause some paused mcp would not update itself until unpaused
+    unhealthy_mcp=$(oc get mcp -o custom-columns=NAME:metadata.name,CONFIG:spec.configuration.name,UPDATING:status.conditions[?\(@.type==\"Updating\"\)].status,DEGRADED:status.conditions[?\(@.type==\"Degraded\"\)].status,DEGRADEDMACHINECOUNT:status.degradedMachineCount --no-headers | grep -v "False.*False.*0")
+    if [[ -n "${unhealthy_mcp}" ]]; then
+        echo "Detected unhealthy mcp:"
+        echo "${unhealthy_mcp}"
+        echo "Real-time detected unhealthy mcp:"
+        oc get mcp -o custom-columns=NAME:metadata.name,CONFIG:spec.configuration.name,UPDATING:status.conditions[?\(@.type==\"Updating\"\)].status,DEGRADED:status.conditions[?\(@.type==\"Degraded\"\)].status,DEGRADEDMACHINECOUNT:status.degradedMachineCount | grep -v "False.*False.*0"
+        echo "Real-time full mcp output:"
+        oc get mcp
+        echo ""
+        unhealthy_mcp_names=$(echo "${unhealthy_mcp}" | awk '{print $1}')
+        echo "Using oc describe to check status of unhealthy mcp ..."
+        for mcp_name in ${unhealthy_mcp_names}; do
+          echo "Name: $mcp_name"
+          oc get mcp $mcp_name -o json || echo "oc get mcp -o json failed"
```

Comment:
As the above prompt is saying "Using oc describe to check status of unhealthy mcp"
so I guess we can keep using `oc describe mcp` here.

I compared the output of `oc describe mcp master` and `oc get mcp master -o json`, seems 
`oc describe` is more useful as it has Events printing out.